### PR TITLE
Constitution update

### PR DESCRIPTION
**Pull request by YaserJazouane**

### CODE REVIEW

**Comment by guibescos** [comment]
I don't know if the pdf belongs here... What if the pdf stops matching the md?

File: docs/constitution/pyth-dao-constitution.md [hunk 1]
```diff
@@ -100,16 +100,16 @@ Any Council member who has voted less than 1/3 of the proposals during the term
...
 
 2. **The Price Feed Council**
 
 The Price Feed Council is made of 7 members who are signers of the Price Feed Multisig Wallet. The Price Feed Multisig Wallet has powers to perform actions that are delegated to it by the Pyth DAO. The execution of such actions by the Price Feed Council requires 5-of-8 approval (with one of the default multisig holders being the Operations Wallet itself). The on-chain submission of a PIP using the Operations Wallet carries one vote in favour of the PIP.
 
-The first election of the Price Feed Council will be ratified on-chain. The date chosen for the first election will form the basis for all future elections. Every election should begin 6 months after the previous election has started and it will replace 3 members of its cohort of 7 members.
+The first election of the Price Feed Council will be conducted via on-chain voting. The date chosen for the first election will form the basis for all future elections. Every election should begin 6 months after the previous election has started and it will replace 3 members of its cohort of 7 members.
 
 The 3 members to remain are selected given the following cascading criteria:
```
Comment (guibescos) [inline docs/constitution/pyth-dao-constitution.md:114]: There's a typo here, 4 members remain

Comment (YaserJazouane) [reply docs/constitution/pyth-dao-constitution.md]: fixed.

**Comment by YaserJazouane** [comment]
> I don't know if the pdf belongs here... What if the pdf stops matching the md?

 it is there to make it easier for people to consume. we can remove after first constitution past.

**Review by jayantk — APPROVED** [review]

**Pull request merged**